src/core/unicode.js

```diff
@@ -65,6 +65,34 @@
     return code;
   }
 
+  function getUnicodeForGlyph(name, glyphsUnicodeMap) {
+    var unicode = glyphsUnicodeMap[name];
+    if (unicode !== undefined) {
+      return unicode;
+    }
+    if (!name) {
+      return -1;
+    }
+    // Try to recover valid Unicode values from 'uniXXXX'/'uXXXX{XX}' glyphs.
+    if (name[0] === 'u') {
+      var nameLen = name.length, hexStr;
+
+      if (name[1] === 'n' && name[2] === 'i' && nameLen === 7) { // 'uniXXXX'
```

Comment:
This isn't in a tight loop or anything, but it's best to do the cheapest check first (nameLen === 7) so the rest are short circuited.